cli/flox/src/commands/install.rs

```diff
@@ -580,13 +678,27 @@ fn add_activation_to_rc_file(
 
 #[cfg(test)]
 mod tests {
+    use flox_rust_sdk::flox::test_helpers::flox_instance;
+    use flox_rust_sdk::models::environment::path_environment::test_helpers::new_path_environment_in;
+    use flox_rust_sdk::models::environment::Environment;
     use flox_rust_sdk::models::lockfile::test_helpers::fake_catalog_package_lock;
     use flox_rust_sdk::models::lockfile::LockedPackageCatalog;
     use flox_rust_sdk::models::manifest::{CatalogPackage, PackageToInstall};
-    use flox_rust_sdk::providers::catalog::SystemEnum;
+    use flox_rust_sdk::providers::catalog::{
+        CatalogClient,
+        Client,
+        MockClient,
+        SystemEnum,
+        GENERATED_DATA,
+    };
+    use flox_test_utils::manifests::EMPTY_ALL_SYSTEMS;
+    use indoc::formatdoc;
+    use nix::libc::REG_EMPTY;
```

Comment:
Note: a bunch of these imports are unnecessary and already cleaned up locally. Trying to avoid another CI run before squashing and merging.